deployments/automated-previews.mdx

```diff
@@ -274,3 +274,161 @@ pipelines:
 <Note>
   The [`--ttl`](/reference/cli#architect-environments-create-environment) flag is used here in order to clean up the environment automatically after one day because BitBucket doesn't have a feature to trigger a pipeline when a pull request is closed.
 </Note>
+
+## CircleCI
+
+CircleCI doesn't have complete features to handle preview environment creation and destruction, but the [Trigger CircleCI Pipeline GitHub Action](https://github.com/CircleCI-Public/trigger-circleci-pipeline-action)
+can help to work around that. The CircleCI GitHub Action can be used in combination with GitHub's event triggers to run a CircleCI workflow.
+
+<Note>
+  This example assumes that `ARCHITECT_EMAIL`, `ARCHITECT_PASSWORD`, `ARCHITECT_ACCOUNT`, and `ARCHITECT_CLUSTER` are set as project variables in CircleCI.
+  Also, the variable `CCI_TOKEN` must be a CircleCI personal access token and be set as a GitHub Action secret in the GitHub repo.
+  `CCI_TOKEN` should _not_ be set to a CircleCI project-specific API token.
+</Note>
+
+<Note>
+  `ARCHITECT_PASSWORD` must be a [personal access
+  token](/deployments/personal-access-tokens).
+</Note>
+
+### Preview environment creation trigger
+
+The file `circleci-create-preview.yml` will trigger a CircleCI workflow to be run when a pull request is created or updated in GitHub. Note that `GHA_Meta` is set to `ARC_CREATE_PREVIEW`
```

Comment:
Does the user need to set `ARC_CREATE_PREVIEW` to anything?

Reply:
No, that's just set so that CircleCI knows which job to run